PrebidMobile/Swift/ConfigurationAndTargeting/Targeting.swift

```diff
@@ -189,6 +189,10 @@ public class Targeting: NSObject {
     /// See CoreLocation framework documentation.
     public var coordinate: NSValue?
     
+    /// Number of decimal places to use when rounding latitude/longitude for device geolocation.
+    /// A value of 0 means no fractional precision (whole degrees only).
+    /// Example: precision 2 ≈ ~1 km accuracy, precision 4 ≈ ~11 m accuracy.
+    public var locationPrecision: UInt = 0
```

Comment:
Using 0 as a Precision value can be confusing. It can mean - no decimal digits. We should use optional here or some other value that does't make sense like a Precision vealue. I prefere optional. But need to be sure that property will be avaliable in the ObjC code.